ports/liblsquic/portfile.cmake

```
@@ -0,0 +1,82 @@
+
+vcpkg_from_github(OUT_SOURCE_PATH SOURCE_PATH
+    REPO litespeedtech/lsquic
+    REF v3.0.4
+    SHA512 9c05045985187ad6e0316b8314dd2e71c2e3b7696210275280d82064509f2a1a0f25928f40212587983eb2c8271e182fd7ca2ac843930e083ff3c5db63820210
+    HEAD_REF master
+    PATCHES
+        export_targets.patch
+        build_tree_version_source.patch)
+
+# Submodules
+vcpkg_from_github(OUT_SOURCE_PATH LSQPACK_SOURCE_PATH
+    REPO litespeedtech/ls-qpack
+    REF v2.2.3
+    HEAD_REF master
+    SHA512 2351cdbaf7f6f1027e9ab814888ad2007f92357f3830ede1dbfb96089cd70a0f85274188b40774208bbb50c6b0400ff04ae0222a2665df9323c6c937227656a6
+)
+if(NOT EXISTS "${SOURCE_PATH}/src/ls-hpack/CMakeLists.txt")
+    file(REMOVE_RECURSE "${SOURCE_PATH}/src/liblsquic/ls-qpack")
+    file(RENAME "${LSQPACK_SOURCE_PATH}" "${SOURCE_PATH}/src/liblsquic/ls-qpack")
+endif()
+
+vcpkg_from_github(OUT_SOURCE_PATH LSHPACK_SOURCE_PATH
+    REPO litespeedtech/ls-hpack
+    REF v2.3.0
+    HEAD_REF master
+    SHA512 45866b18042125cbbd008eed2935a938a42e1682030aa52ff4a324ddbad7bf9bd483161352cc8988bae668e132ee8b4b043ddc09d9e0316a66aaefd927ae2d76
+)
+if(NOT EXISTS "${SOURCE_PATH}/src/lshpack/CMakeLists.txt")
+    file(REMOVE_RECURSE "${SOURCE_PATH}/src/lshpack")
+    file(RENAME "${LSHPACK_SOURCE_PATH}" "${SOURCE_PATH}/src/lshpack")
+endif()
+
+# Configuration
+if(WIN32)
+  # The lib uses CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS, at least until
+  # https://github.com/litespeedtech/lsquic/pull/371 or similar is merged
+  vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+endif()
+vcpkg_find_acquire_program(PERL)
+get_filename_component(PERL_EXE_PATH ${PERL} DIRECTORY)
+vcpkg_add_to_path(${PERL_EXE_PATH})
```

```suggestion
get_filename_component(PERL_EXE_PATH "${PERL}" DIRECTORY)
vcpkg_add_to_path("${PERL_EXE_PATH}")
```
Please also add double quotes to all absolute paths.